turbo-rails.gemspec

```diff
@@ -9,10 +9,10 @@ Gem::Specification.new do |s|
   s.homepage = "https://github.com/hotwired/turbo-rails"
   s.license  = "MIT"
 
-  s.required_ruby_version = ">= 2.6.0"
+  s.required_ruby_version = ">= 3.1.0"
```

Comment:
Thank you for opening this PR.

There is a fair amount of overlap with https://github.com/hotwired/turbo-rails/pull/698, which also changes the Ruby version requirement.

As it relates to https://github.com/hotwired/turbo-rails/issues/681, I think there still needs to be a final release that deprecates support for 2.6 before `main` can drop support.